guides/source/getting_started.md

```diff
@@ -242,262 +241,740 @@ different _actions_. Each action's purpose is to collect information to provide
+used in this request. The `layouts/application` mentioned here is located at
+`app/views/layouts/application.html.erb`. This file contains the following
+content:
+
+```erb
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Blog</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+    <%= yield %>
+  </body>
+</html>
+```
+
+This file is the common bits of HTML that will be wrapped around whatever our
+views return. The view's output goes right inside that `<body>` tag, where `<%=
+yield %>` is.
+
+The next line is:
+
+```
+[Webpacker] Everything's up-to-date. Nothing to do
+```
+
+Webpacker is a gem that comes with Rails that manages our assets. Right now, we
+don't have any assets and so we can ignore this line, and Webpacker in general.
+
+The final line is this:
+
+```
+Completed 200 OK in 10ms (Views: 7.9ms | ActiveRecord: 0.0ms | Allocations: 4989)
+```
+
+It tells us that the request completed successfully. We know it's successful
+because we saw it ourselves, but also because this line contains `200 OK`,
+which is the HTTP status code for successful responses. The rest of the
+information on this line is performance metrics.
+
+We have now finished rendering our first view. We created two routes that go to
+the same action -- `GET /` and `GET /articles`. That action is the `index`
+action in the `ArticlesController`. When we make a request to either of these
+paths, the `app/views/articles/index.html.erb` view is rendered, wrapped in the
+`app/views/layouts/application.html.erb` layout.
+
 TIP: For more information about routing, refer to [Rails Routing from the Outside In](routing.html).
 
-Getting Up and Running
-----------------------
+## Adding Articles
+
+So far, we have seen routes, controllers, actions and views within our Rails
+application. All of these are conventional parts of Rails applications and it
+is done this way to follow the MVC pattern. The MVC pattern is an application
+design pattern which makes it easy to separate the different responsibilities
+of applications into easy to reason about pieces.
 
-Now that you've seen how to create a controller, an action, and a view, let's
-create something with a bit more substance.
+So with "MVC", you might guess that the "V" stands for "View" and the "C"
+stands for controller, but you might have trouble guessing what the "M" stands
+for. This next section is all about that "M" part, the _model_.
 
-In the Blog application, you will now create a new _resource_. A resource is the
-term used for a collection of similar objects, such as articles, people, or
-animals.
-You can create, read, update, and destroy items for a resource and these
-operations are referred to as _CRUD_ operations.
+### Generating an Article model
```

Comment:
According to the guidelines: 'When writing headings, capitalize all words except for prepositions, conjunctions, internal articles, and forms of the verb "to be"'
https://edgeguides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#headings

So I think `model` should be capitalized.